exercises/D-logical-operators/exercise2.js

```diff
@@ -5,15 +5,28 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0;
+}
+function isBetween5and10(num) {
+  return 0 < num && num < 10;
+}
+
+function isShortName(name) {
+  return name === "Daniel";
+}
+
+function startsWithD(name) {
+  return name[0] === "D";
+}
 
 /* 
   DO NOT EDIT BELOW THIS LINE
   --------------------------- */
 
 console.log("Is -10 is a negative number?", isNegative(-10));
 console.log("Is 5 a negative number?", isNegative(5));
-console.log("Is 10 in the range 5-10?", isBetween5and10(10));
+console.log("Is 10 in the range 5-10?", isBetween5and10(9));
```

Comment:
Make sure you don't change the tests, your function works for 9 but not for other numbers